Gemfile

```diff
@@ -1,7 +1,22 @@
-source :rubygems
+source 'https://rubygems.org'
 
-gemspec
+gem 'rake'
+gem 'rspec-puppet'
+gem 'puppet-lint'
+gem 'puppet-syntax'
+gem 'mocha'
 
+group :development do
+  gem 'yard'
+  gem 'pry'
+  gem 'jeweler'
+  gem "puppet", ENV['PUPPET_VERSION'] || '~> 3.8.3'
+end
+
+group :test do
+end
+
+# I don't know the purpose of this below
```

Comment:
This was so someone could add more gems by making a .local file... I'd probably just delete this stuff